_drafts/2019-06-03-invariant-intro.md

```diff
@@ -13,16 +13,17 @@ Target audience: Technical, CTOs, programming expertise (non-functional), Datalo
 Novelty: describe concept well enough for Datalog interests
 -->
 
-<blockquote class="literal left"> "The principle of least action is the basic
+# Goals
+{% quote Principle of Least Action, Scholarpedia|http://www.scholarpedia.org/article/Principle_of_least_action) %}
```

Comment:
I find this the best example to capture the importance of invariants in constraining the dynamics of the world, but maybe it is too academic. It is definitely not something that is well-known by computer scientists.